Add information to create a database named 'daprStore' in case no dat… (#1629)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md

```

> 
> You can then interact with the server using `localhost:27017`.
> 
> {{% alert title="Note" %}}
> Please create a database named `daprStore` in case the `databaseName` is not specified in the component definition. 
> {{% /alert %}}
> 

Reviewer: ```suggestion

You can then interact with the server using `localhost:27017`.

If you do not specify a `databaseName` value in your component definition, make sure to create a database named `daprStore`. 

```

Author: Sure, made the necessary change.
